WORKSPACE

```diff
@@ -109,4 +112,25 @@ maven(artifacts)
 load("@graknlabs_bazel_distribution//common:rules.bzl", "workspace_refs")
 workspace_refs(
     name = "graknlabs_protocol_workspace_refs"
-)
\ No newline at end of file
+)
+
+#########################
+# Load NPM dependencies #
+#########################
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+http_archive(
```

Comment:
Can we update the version of `rules_nodejs` in `@graknlabs_dependencies` with this one and then have the `WORKSPACE` file load from there instead? You can get @alexjpwalker 's help for the know-how.